docs/make.jl

```diff
@@ -35,29 +35,33 @@ makedocs(
     # Explicitly specify documentation structure
     pages = [
         "Home" => "index.md",
-        "Visualization" => "visualization.md",
-        "Conventions" => [
-            "User-facing conventions" => "conventions_user.md",
-            "Internal conventions" => "conventions_internal.md",
+        "Getting started" => [
+            "Overview" => "overview.md",
+            "Visualization" => "visualization.md",
         ],
-        "Meshes" => [
-            "Tree mesh" => joinpath("meshes", "tree_mesh.md"),
-            "Structured mesh" => joinpath("meshes", "structured_mesh.md"),
-            "Unstructured mesh" => joinpath("meshes", "unstructured_quad_mesh.md"),
-        ],
-        "Time integration" => "time_integration.md",
-        "Callbacks" => "callbacks.md",
-        "Development" => "development.md",
-        "Performance" => "performance.md",
-        "Parallelization" => "parallelization.md",
-        "Testing" => "testing.md",
         "Tutorials" => [
             "Adding a new equation" => "adding_a_new_equation.md",
             "Differentiable programming" => "differentiable_programming.md",
         ],
-        "Troubleshooting" => "troubleshooting.md",
-        "Style guide" => "styleguide.md",
-        "GitHub & Git" => "github-git.md",
+        "Basic building blocks" => [
+            "Meshes" => [
+                "Tree mesh" => joinpath("meshes", "tree_mesh.md"),
+                "Structured mesh" => joinpath("meshes", "structured_mesh.md"),
+                "Unstructured mesh" => joinpath("meshes", "unstructured_quad_mesh.md"),
+            ],
+            "Time integration" => "time_integration.md",
+            "Callbacks" => "callbacks.md",
+        ],
+        "Advanced topics and developer documentation" => [
```

Comment:
```suggestion
        "Advanced topics & developers" => [
```
I don't know if this is really a good title, but I'd really prefer it if we can reduce the title to a single line:
![image](https://user-images.githubusercontent.com/3637659/115712226-c87b7500-a374-11eb-9c9c-f2f7afa7d727.png)